databricks-iris/{{ cookiecutter.repo_name }}/conf/base/logging.yml

```
@@ -0,0 +1,32 @@
+version: 1
+
+disable_existing_loggers: False
+
+formatters:
+  simple:
+    format: "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+handlers:
+  info_file_handler:
+    class: logging.handlers.RotatingFileHandler
+    level: INFO
+    formatter: simple
+    filename: /dbfs/FileStore/iris-databricks/logs/info.log
```

```suggestion
    filename: /dbfs/FileStore/iris-databricks/info.log
```

We are not using `logs` folder anymore

---

Done.